src/cli/index.ts

```diff
@@ -163,11 +167,30 @@ async function run() {
     }
   );
 
+  function toLeopard() {
+    const { leopardUrl: leopardURL } = options;
```

Comment:
Not particularly important, but is there any reason why this isn't simplified?

```diff
- const { leopardUrl: leopardURL } = options;
+ const { leopardUrl } = options;
```

Reply:
Oh, yes! I didn't even recognize that there was a rename happening here. I read it (incorrectly) as `{ leopardUrl: leopardUrl }`.